### ISSUE TITLE
Video 

### ISSUE THREAD

**Issue by RGB-123** [scanlime/led-cartography#1]
M, any chance you are thinking of shooting a video of how this works?  I am guessing it would be a pretty awesome demo.  Ryan

**Comment by scanlime** [comment]
Good idea. I don't have a good demo for this at the moment- it's for a
project that's being fabricated far away, and the smaller models I've been
using to debug the code are fairly boring by comparison. Soon though!
